microcosm_caching/memcached.py

```diff
@@ -69,15 +63,13 @@ def __init__(
         servers: Optional[Tuple[str, int]] = None,
         connect_timeout=None,
         read_timeout=None,
-        serializer=json_serializer,
-        deserializer=json_deserializer,
+        serde=None,
         testing=False,
     ):
         client_kwargs = dict(
             connect_timeout=connect_timeout,
             timeout=read_timeout,
-            serializer=serializer,
-            deserializer=deserializer,
+            serde=serde if serde is not None else JsonSerde(),
```

Comment:
Is it worth adding tests that `MemcachedCache` can:

1. use a sensible default value
1. use a different serializer if one is provided